phpBB/phpbb/textformatter/s9e/parser.php

```diff
@@ -56,24 +50,20 @@ public function __construct(\phpbb\cache\driver\driver_interface $cache, $key, \
 
 		$this->dispatcher = $dispatcher;
 		$this->parser = $parser;
-		$this->user = $user;
 		$parser = $this;
 
 		/**
 		* Configure the parser service
 		*
 		* Can be used to:
-		*  - toggle features according to the user's preferences,
-		*  - toggle BBCodes according to the user's permissions,
```

Comment:
I don't see the user object being used in the class for anything other than loading lang keys in get_errors() below.

Why does this event need to pass the $user object in an event?

ABBC ext (3.2) already uses this event to toggle bbcodes for a user based on their group membership. The $user is handled independently within the ABBC3 extension's own methods.

I may not be seeing the bigger picture but I'm not sure how the user object here serves any purpose in this event.